website/docs/guides/adapter-creation.md

```diff
@@ -11,6 +11,8 @@ level: 'Advanced'
 recently_updated: true
 ---
 
+# test h1 title <Lifecycle status="team,enterprise"/>
```

Comment:
went ahead and tested this Lifecyle use case too

Reply:
Nice catch @nghi-ly! I can get that updated tomorrow. I'm pulling in the HTML from the badge into the custom h1 section we're using for guides, so I should just have to update the part of the script grabbing that code to grab all badges!